php/class-wp-cli.php

```diff
@@ -1263,7 +1263,7 @@ public static function get_config( $key = null ) {
 	}
 
 	/**
-	 * Run a WP-CLI command.
+	 * Run a WP-CLI command. Optionally modify the context in which the command runs and how the result is processed.
```

Comment:
This might be somewhat redundant with an $options array documented, but no strong opinion.

Reply:
This is indeed redundant. Moreover summary is generally one sentence. You can update description down below if needed.